### ISSUE TITLE
Light theme does not apply

### ISSUE THREAD

**Issue by krystalcode** [purcell/color-theme-sanityinc-solarized#30]
I have this installed via your `.emacs.d` repository.

When I do `M-x color-theme-sanityinc-solarized-dark`, here's what I get:
![Screenshot_20241122_004353](https://github.com/user-attachments/assets/8e7ee3c6-c922-4bc9-a11a-856c8986509e)

Then I do `M-x color-theme-sanityinc-solarized-light`, here's what I get:
![Screenshot_20241122_004410](https://github.com/user-attachments/assets/3fd7880d-1da0-4413-ba42-80a06955a2ff)

This does not look like the light theme that I'd expect, with the creamy background. How can I troubleshoot that? I don't see anything relevant in `*Messages*`.

**Comment by purcell** [comment]
I made some changes in f42431850e0ff0cff90c6cc39edc222faa40323d to align with some improvements I made a while back in the `tomorrow` themes, so perhaps give that a try and see if it helps. It works for me locally, fwiw.

**Comment by krystalcode** [comment]
@purcell Thanks for the response, I'll try that.

I suspect that the problems that I have are due to running emacs within a container i.e. some graphical capabilities might not be fully enabled, and moreover I start emacs as a server and then start frames using `emacsclient -t` which causes an error saying that the window system not being available.

Even though I'm not sure, if I set the background color manually, for example, it works.

**Comment by purcell** [comment]
Yeah, I wouldn't expect those factors to affect it, but you never know. :)
